spec/controllers/user/teams_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+require 'user/teams_controller'
```

Comment:
Try to create the `User::TeamsController` like this in the `controllers/user/teams_controller.rb` file:

```
module User
  class TeamsController
    # ...
  end
end
```

instead of `User::TeamsController` and then remove this require to see if it works.